doc/man7/provider-signature.pod

```diff
@@ -234,6 +300,34 @@ is I<tbslen> bytes long.
 The signature is pointed to by the I<sig> parameter which is I<siglen> bytes
 long.
 
+=head2 Message Verify Functions
+
+These functions are suitable to algorithms that implement algorithms that
```

Comment:
We sill have algorithms that implement algorithms :-)

Reply:
Fixed